### PR TITLE
Create update.bat

### PR DESCRIPTION
**Pull request by BrandonStaab**
Short, sweet and to the point way of updating based off of pahi's example.

### CODE REVIEW

File: update.bat [hunk 1]
```diff
@@ -0,0 +1 @@
+gradle.bat clean && gradle.bat setupdecompworkspace && gradle.bat idea
```
Comment (TTF5) [inline update.bat:1]: I think it is gradlew.bat and not gradle.bat :D But it's a good idea

**Comment by pahimar** [comment]
The same result can be made with 'gradlew clean setupDecompWorkspace idea'

However I will not be merging this for a number of reasons;
1) This is a Windows only option
2) It pre-supposes that the person is using IntelliJ

**Comment by Parker8283** [comment]
Also, don't be SOO lazy that you can't run some simple commands from the command line.

**Comment by ghost** [comment]
If you **are** so lazy, create it for yourself, not for everybody.

**Pull request closed without merging**